library/formatting.inc.php

```diff
@@ -9,7 +9,7 @@
 function oeFormatMoney($amount, $symbol = false)
 {
     $s = number_format(
-        $amount,
+        doubleval($amount),
```

Comment:
Would use floatval instead
(since doubleval is simply a alias of floatval, might as well go to the source :) )
(and just let me know if this makes the warning go away; then will bring it into the codebase)